services/wallet/balance_test.go

```diff
@@ -61,13 +61,13 @@ func (s *BalancesSuite) SetupTest() {
 	}
 }
 
-func (s *BalancesSuite) TestBalanceEqualPerToken() {
+/*func (s *BalancesSuite) TestBalanceEqualPerToken() {
```

Comment:
Why are we commenting out this code?

If this test is failing, we should find the reason and fix it.